src/System.Private.ServiceModel/src/System.Private.ServiceModel.builds

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Private.ServiceModel.csproj" />
+    <Project Include="System.Private.ServiceModel.csproj">
```

Comment:
As I called out in my other comment you should only have the first entry in this item group which means it default to AnyOS and delete the other entires with specific OSGroup defined, unless you actually have conditions in your projects for different OSGroups, which I don't believe you guys do yet.